coordinator/tests/services/translation/request/agent-unattended.spec.ts

```diff
@@ -67,18 +67,18 @@ describe("convertOrganization", () => {
     expect(mockConvertTelecom).toBeCalledWith(testData.organization.telecom[0], "Organization.telecom")
   })
 
-  test("Converts organization correctly if organization is missing telecom", () => {
-    const org2 = testData.organization
-    delete org2.telecom
-    convertOrganization(org2, testData.telecom)
+  test("Uses passed in telecom if organization doesn't have one", () => {
+    const testOrganization = testData.organization
+    delete testOrganization.telecom
```

Comment:
I'd be a bit cautious of doing this delete here as I think on line 71 you will have passed a reference so this `delete` will be modifying the original imported `testData.organization` and can cause issues if when changing the order of tests

The safest way to do things like this is something like:

```
const testOrganization = {
    ...testData.organization,
    telecom: null
}
```